### ISSUE TITLE
`add_sd_*` is drawing at twice the actual SD value

### ISSUE THREAD

**Issue by awata25** [jbengler/tidyplots#25]
Hello! Thank you for developing such an easy-to-use package.
When drawing the graphs with this package I noticed that the standard deviation values are double of the actual ones. For example, when I draw a bar graph with the following code: 

```
time_course |>
  dplyr::filter(day == "15") |>
  tidyplot(x = treatment, y = score, dodge_width = 0) |>
  add_mean_bar(alpha = 0.3) |>
  add_data_points() |>
  add_sd_errorbar()
```
![time_course_bar-and-dots](https://github.com/user-attachments/assets/61986255-2131-4da5-9074-b37c64986ee4)

The SD of untreated looks like around 1.5, but when I calculate it:

```
> time_course |>
+   dplyr::filter(day == "15") |>
+   summarise(SD = sd(score, na.rm = TRUE),
+             .by =treatment)
# A tibble: 2 × 2
  treatment    SD
  <chr>     <dbl>
1 untreated 0.877
2 treated   1.17 
```

The actual SD value is 0.877. This phenomenon also occurs with `add_sd_ribbon`.
I think it is because Hmisc::smean.sdl() is used in the function definition. Hmisc::smean.sd() would not double the SD and would draw without problems.

**Comment by jbengler** [comment]
Hi @awata25 

Thank you for using tidyplots and for spotting this *important* bug!
I just pushed a fix to the development version.

You can check it by installing tidyplots like so:

```
install.packages("devtools")
devtools::install_github("jbengler/tidyplots")
```

Best
Jan

**Comment by jbengler** [comment]
Hi @awata25 

Thanks again for spotting this bug!

I just released the fixed version tidyplots 0.2.0 on CRAN. 

Best
Jan

**Comment by awata25** [comment]
Thank you @jbengler ! I will update it and use it in the future ;)
